src/zoomies.cpp

```diff
@@ -463,9 +464,9 @@ int flight_test(struct char_data *ch, struct veh_data *veh) {
       break;
   }
   
-  int dice = get_skill(ch, skill_num, tn, rbuf, sizeof(rbuf));
+  int dice = get_autopilot + get_skill(ch, skill_num, tn, rbuf, sizeof(rbuf));
   int successes = success_test(dice, tn, ch, "flight-test");
-  snprintf(ENDOF(rbuf), sizeof(rbuf) - strlen(rbuf), ". Dice %d, TN %d: %d success%s.", dice, tn, successes, successes == 1 ? "" : "es");
+  snprintf(ENDOF(rbuf), sizeof(rbuf) - strlen(rbuf), ". Dice %d, Autopilot Rating %d, TN %d: %d success%s.", dice, get_autopilot, tn, successes, successes == 1 ? "" : "es");
```

Comment:
This will be confusing for players since 'dice' already contains the autopilot rating. Separate out the skill dice (int skill = x, int pilot = y, int successes = success_test(x + y, tn...)) and report on it that way to avoid confusion.

Reply:
Should be easier to digest

Reply:
This actually won't work, because 467's `get_skill(ch, skill_num + get_autopilot, ...)` means that the skill rolled will vary based on what your autopilot rating is. For instance, if you're driving a truck (`SKILL_PILOT_TRUCK (63)`) with autopilot 5, you will get the dice for your skill in `63 + 5` aka `SKILL_DANCING (68)`.

Do this instead:
```c++
int dice = get_skill(ch, skill_num, tn, rbuf, sizeof(rbuf));
int successes = success_test(dice + get_autopilot, tn, ch, "flight-test");
snprintf(ENDOF(rbuf), sizeof(rbuf) - strlen(rbuf), ". Skill %d, Autopilot Rating %d, TN %d: %d success%s.", dice, get_autopilot, tn, successes, successes == 1 ? "" : "es");
```

Reply:
We didn't realize that Vehicle Code was the real End Game Boss all along. Innocence has been lost.